containers/zammad/docker-entrypoint.sh

```diff
@@ -54,7 +54,9 @@ if [ "$1" = 'zammad-init' ]; then
   cd "${ZAMMAD_DIR}"
 
   # configure database
-  sed -e "s#.*adapter:.*#  adapter: postgresql#g" -e "s#.*database:.*#  database: ${POSTGRESQL_DB}#g" -e "s#.*username:.*#  username: ${POSTGRESQL_USER}#g" -e "s#.*password:.*#  password: ${POSTGRESQL_PASS}\\n  host: ${POSTGRESQL_HOST}\\n  port: ${POSTGRESQL_PORT}#g" < contrib/packager.io/database.yml.pkgr > config/database.yml
+  # https://stackoverflow.com/questions/407523/escape-a-string-for-a-sed-replace-pattern
+  ESCAPED_POSTGRESQL_PASS=$(echo "$POSTGRESQL_PASS" | sed -e 's/[\/&]/\\&/g')
```

Comment:
Thats not a generic solution for special characters as it will only fetch "&".
Other charactes could be problematic too.
I would not like to implement it this way.

Reply:
@monotek thanks for your feedback. I checked this again based on the [linked in-depth article](https://stackoverflow.com/questions/407523/escape-a-string-for-a-sed-replace-pattern), which says:

> The replacement string in a sed s/// command is not a regex, but it recognizes placeholders that refer to either the entire string matched by the regex (&) or specific capture-group results by index (\1, \2, ...), so these must be escaped, along with the (customary) regex delimiter, /.

This is exactly what the code at hand does, it handles `/`, `\` and `&`. Therefore I would believe that all relevant cases are properly handled and that we could proceed with merging this.